params/protocol_params.go

```diff
@@ -16,7 +16,18 @@
 
 package params
 
-import "math/big"
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var (
+	// The base fee portion of the transaction fee accumulates at this predeploy
+	OptimismBaseFeeRecipient = common.HexToAddress("0x4200000000000000000000000000000000000019")
```

Comment:
Need to be put in the table of predeploys in the `specs` folder of the monorepo.

Reply:
There is a PR that does this